ouroboros-network/test/Test/Pipe.hs

```diff
@@ -27,7 +28,7 @@ import           Control.Tracer
 import qualified Network.Mux.Bearer.Pipe as Mx
 import           Ouroboros.Network.Mux
 
-import           Ouroboros.Network.Block (encodeTip, decodeTip)
+import           Ouroboros.Network.Block (decodeTip, encodeTip)
```

Comment:
related?

Reply:
No, just inconsistent use of the formatter in this repo.